client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx

```diff
@@ -754,18 +780,44 @@ export default function CampaignItemDetails( props: Props ) {
 						{ shouldShowStats && (
 							<div className="campaign-item-details__main-stats-container">
 								<div className="campaign-item-details__main-stats campaign-item-details__impressions">
+									{ duration_days && (
+										<div className="campaign-item-details__main-stats-row ">
+											<div>
+												<span className="campaign-item-details__label">
+													{ translate( 'Duration' ) }
+												</span>
+												<span className="campaign-item-details__text wp-brand-font">
+													{ ! isLoading ? durationFormatted : <FlexibleSkeleton /> }
+												</span>
+											</div>
+											<div>
+												<span className="campaign-item-details__label">
+													{ translate( 'Run between' ) }
+													&nbsp;
+													<InfoPopover position="right">
+														<span className="popover-title">
+															{ ! isLoading ? durationDateAndTimeFormatted : <FlexibleSkeleton /> }
+														</span>
+													</InfoPopover>
+												</span>
+												<span className="campaign-item-details__text wp-brand-font">
+													{ ! isLoading ? durationDateFormatted : <FlexibleSkeleton /> }
+												</span>
+											</div>
+										</div>
+									) }
 									<div className="campaign-item-details__main-stats-row ">
 										<div>
 											<span className="campaign-item-details__label">
-												{ translate( 'Clicks' ) }
+												{ translate( 'Visitors' ) }
```

Comment:
If I am not wrong, we decided to keep using clicks because they are a different metric than Visitors (not all clicks will end up being visitors). We didn't want to confuse users when they see visitors on the Stats page.

It may have changed since my AFK, but in the widget, we still use the clicks:
![Screenshot 2024-12-12 at 11 18 21 AM](https://github.com/user-attachments/assets/3d4a578e-e58b-42d2-a8af-25eca52a5efd)

The designs are not fully updated, but you can talk with Bart about this. I don't have the design P2 at hand, but I remember there was a discussion about this.